reliquery/storage.py

```diff
@@ -505,6 +510,458 @@ def get_all_relic_data(self) -> List[Dict]:
             return relic_data
 
 
+class GoogleDriveStorage(Storage):
+    def __init__(
+        self,
+        prefix: str,
+        name: str,
+        token_file: str,
+        SCOPES: list,
+        shared_folder_id: str,
+    ):
+        self.prefix = prefix
+        self.name = name
+        self.token_file = token_file
+        self.SCOPES = SCOPES
+        self.shared_folder_id = shared_folder_id
+
+        creds = None
+        if os.path.exists(token_file):
+            creds = service_account.Credentials.from_service_account_file(
+                token_file, scopes=SCOPES
+            )
+        try:
+            self.service = build("drive", "v3", credentials=creds)
+        except IOError:
+            raise StorageItemDoesNotExist
+
+        # Check for prefix folder
+        results = (
+            self.service.files()
+            .list(
+                q="parents in '{}'".format(self.shared_folder_id),
```

Comment:
python "f" strings are the modern thing here.

Reply:
since format is used everwhere, don't bother changing it here, but just in the future something to be aware of.